### PR TITLE
Add hover effect in simulator button

### LINKED ISSUE #1105: Add hover effect on simulator button
**Describe the bug**
Adding hover effect on simulator button

1. Go to homepage
2. See error

**Screenshots**
ezgif com-video-to-gif

**Are you working on this issue? **
Yes

### PR DESCRIPTION
**Pull request by rishipurwar1**
Fixes #1105 

#### Describe the changes you have made in this PR -
Add a hover effect in simulator button

### Screenshots of the changes -
Before

bug#1105

After
ezgif com-video-to-gif (3)

### CODE REVIEW

**Comment by darkshredder** [comment]
looks cool.....but I think you should change the names of css like dont name it button be little specific

**Comment by rishipurwar1** [comment]
> looks cool.....but I think you should change the names of css like dont name it button be little specific
I'll fix that.

**Comment by rishipurwar1** [comment]
@darkshredder
Please tell me how I resolve "line contains trailing whitespace" error?

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -88,3 +88,47 @@
...
+  border: 2px solid #42b983;
+  box-sizing: border-box;
+  cursor: pointer;
+  float: left;
+  height: 50px;
+  margin: 0;
+  overflow: hidden;
+  position: relative;
+  text-align: center;
+  width: 120px;
```
[the remark this answers is not in the document]
Comment (darkshredder) [reply app/assets/stylesheets/layout.scss]: @rishipurwar1  on 102 line remove spaces at end of ";"

[the remark this answers is not in the document]
Comment (darkshredder) [reply app/assets/stylesheets/layout.scss]: > @rishipurwar1 on 102 line remove spaces at end of ";"

Do this

**Comment by darkshredder** [comment]
@rishipurwar1 on 102 line remove spaces at end of ";"

**Comment by rishipurwar1** [comment]
> @rishipurwar1 on 97 line remove spaces after "}" on the same line

Thanks @darkshredder 
I have one more error "Merge rule .simulatorButton a with rule on line 92"
Please help me with this error.

**Comment by rishipurwar1** [comment]
Please review my pr @tachyons @nitin10s @aayush-05

**Comment by rishipurwar1** [comment]
Ok, I will fix that

**Comment by rishipurwar1** [comment]
Please review @nitin10s

**Comment by rishipurwar1** [comment]
Fix simulator text alignment. Please check @nitin10s

**Comment by aayush-05** [comment]
How are the changes in home.scsss and index.html related to this fix ? Also, please fix merge conflicts.

**Comment by rishipurwar1** [comment]
I made changes in layout.scss and header.html.
And there is no merge conflict.

**Comment by rishipurwar1** [comment]
Fixed merging conflict and I made changes in layout.scss and header.html.
Please check @aayush-05

File: app/assets/stylesheets/layout.scss [hunk 2]
```diff
@@ -107,3 +107,48 @@
...
+  background: $green;
+  height: 36px;
+  left: -140px;
+  position: absolute;
+  transition: all .5s ease;
+  width: 94px;
+  z-index: 1;
+}
+
+.simulator-button:hover .effect {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: use sass syntax

File: app/assets/stylesheets/layout.scss [hunk 3]
```diff
@@ -107,3 +107,48 @@
...
+  left: 0;
+}
+
+.simulator-button:hover a {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: use sass syntax

**Comment by rishipurwar1** [comment]
Hello @tachyons 
In this file, all the code is written in CSS. So, what is the need for writing this block of code in sass?

**Comment by aayush-05** [comment]
@rishipurwar1 read about sass pseudo selectors

**Comment by rishipurwar1** [comment]
I am woorking on this @nitin10s

**Comment by nitin10s** [comment]
You can once check #1226

**Comment by aayush-05** [comment]
@tachyons this can be closed, #1226 will be merged after required changes

**Comment by aayush-05** [comment]
@nitin10s can this be closed ?

**Comment by nitin10s** [comment]
Are you working on this @rishipurwar1 ?

**Comment by aayush-05** [comment]
@tachyons @nitin10s this can be closed >> fixed with new UI

**Pull request closed without merging**